internal/install/_static/terraform_deployer_run.sh

```diff
@@ -2,25 +2,41 @@
 
```

Comment:
I don't see any container logs for the GCP test package in `beats-ci-temp-internal/Ingest-manager/elastic-package/PR-662-16/insecure-logs/gcp`. Is it intended?

Reply:
No is not, [from the pipeline logs](https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Felastic-package/detail/PR-662/16/pipeline/280#step-1190-log-244) it seems it not running system tests (command start at line 244, hidden by the task header)

I set up the `env.yml` file in [`data_stream/billing/_dev/deploy/tf`](https://github.com/elastic/elastic-package/pull/662/files#diff-344f9471070f52db55ee64eb5349532ba4f52bcdf2a2418bfa983add481ef6d6). Is this not enough to run system tests?

Reply:
I checked your branch and you haven't configured any test policies. Test policies are used by elastic-package during system tests. Otherwise the elastic-package won't know what you re trying to test. It's covered in [our manual](https://github.com/elastic/elastic-package/blob/main/docs/howto/system_testing.md#test-case-definition).

Please take a look at the AWS test package and [ec2_metrics](https://github.com/endorama/elastic-package/tree/terraform-deployer-fix-gcp-auth/test/packages/parallel/aws/data_stream/ec2_metrics/_dev/test/system) tests.

Reply:
Test package with first test case (not requiring changes in this PR) added in https://github.com/elastic/elastic-package/pull/701